docs/django-admin/roles.md

```diff
@@ -9,6 +9,30 @@ our `user_group` model and run in a migration.
 
 For more details, refer to the [user group model](../../src/registrar/models/user_group.py).
 
+## Adding a user as analyst or granting full access via django-admin
+
+If a new team member has joined whose data is NOT included in fixtures,
```

Comment:
I don't think analysts will know what fixtures is or what it matters. Can a short description and/or link be added?